### DISCUSSION TITLE
Suppressing the Output of the Starting Command in stderr

### DISCUSSION THREAD

**Discussion by jskorlol** [oven-sh/bun#6246] [Q&A] [answered]
When executing the command bun run test-execute-script --silent --no-update-notifier, the following message is logged in stderr:

```
$ bun scripts/test-execute-script.js --silent --no-update-notifier
```

I would like to suppress this starting command message from being logged in stderr. Is there a way to achieve this?

In Node.js, using the --silent flag suppressed the output of the execution command. Is there an equivalent option or workaround available in bun?

**Comment by Jarred-Sumner** [comment, marked as answer]
bun flags must go first

`bun --silent scripts/test-execute-script.js --no-update-notifier`

**Reply by buzzy** [reply]
Why does --silent still print Bug Inspector messages?

```
bun --silent --hot --inspect=ws://0.0.0.0:19229/debug run server.ts
--------------------- Bun Inspector ---------------------
Listening:
  ws://0.0.0.0:19229/debug
Inspect in browser:
  [https://debug.bun.sh/#0.0.0.0:19229/debug⁠](https://debug.bun.sh/#0.0.0.0:19229/debug)

--------------------- Bun Inspector ---------------------
```
